Add label filtering to CTS (#28)
kyma-incubator/octopus: docs/crd-cluster-test-suite.md

| **metadata.name** |    **YES**   | Specifies the name of the CR. |
| **spec.selectors** | **NO** | Defines which tests should be executed. You can define tests by specifying their names or labels. Selectors are additive. If not defined, all tests from all Namespaces are executed.
| **spec.selectors.matchNames** | **NO** | Lists TestDefinitions to execute. For every element on the list, specify **name** and **namespace** that refers to a TestDefinition. |

> | **spec.selectors.matchLabelExpressions** | **NO** | Lists of label expressions that match labels of TestDefinitions to execute. A TestDefinition is selected if at least one label expression matches. For the complete grammar see: https://kubernetes.io/docs/concepts/overview/working-with-objects/labels | 

Reviewer: ```suggestion
| **spec.selectors.matchLabelExpressions** | **NO** | Lists label expressions that match labels of TestDefinitions to execute. A TestDefinition is selected if at least one label expression matches. See [this](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels) document for more details. | 
```
